data/recreation/local-parks/index.html

```diff
@@ -8,9 +8,9 @@
 date: 2011-11-22 15:04:53 -0700
 categories: []
 LocalParks:
-  downloads:
-    - url: https://drive.google.com/drive/folders/0ByStJjVZ7c7mdm1nUTl5dVFodXM
-      label: Local Park Package
+  hub:
+    name: Utah Parks Local
+    item_id: 818536733987486b8082e4c5dec0a1dd
   updates:
     - January 2020
   endpoints:
```

Comment:
**nitpick**: delete this endpoints section as that is duplicated by the hub section